constellations/gemini-user/src/announcements/announce.sass

```diff
@@ -0,0 +1,19 @@
+@import "../utils/_constants.sass"
+
+#view-announce
+  h1
+    width: 100%
+    color: $colorTurquoise
+    text-align: center
+    margin: 90px 0px 50px
```

Comment:
Be sure to adhere to CSS ordering: https://github.com/ahsu1230/mathnavigatorSite/blob/master/resources/frontend/css_ordering.md

Place margin under width. Then text-align, then color.

Reply:
And add a new line after the last property.